src/docfx/lib/JsonUtility.cs

```diff
@@ -184,6 +184,12 @@ public static JObject Merge(JObject first, IEnumerable<JObject> objs)
             return result;
         }
 
+        public static JObject ValidateNullValue(this JObject graph)
```

Comment:
`JToken RemoveNulls(JToken)`

Reply:
+1

---
In reply to: [211813607](https://github.com/dotnet/docfx/pull/3252#discussion_r211813607) [](ancestors = 211813607)